scripts/prototype.py

```diff
@@ -1,17 +1,10 @@
-from validitysensor.tls import tls
-from validitysensor.usb import usb
-from validitysensor.db import db
-from validitysensor.flash import read_flash
-from validitysensor.sensor import *
-from validitysensor.sid import *
```

Comment:
In this file all these imports, (including `*`) were actually intended.

The only purpose of this file is to provide a nice REPL command line with everything imported and ready to use by doing something like this:
```
$ python3
Python 3.6.7 (default, Oct 22 2018, 11:32:17) 
[GCC 8.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from prototype import *
>>> open9x()
>>>
```

It is a very convenient tool when hacking things.

Reply:
Ah, okay, I will reverse those changes for that file